conf/modules.config

```diff
@@ -51,23 +51,20 @@ process {
             enabled: params.save_reference
         ]
     }
-    // withName: 'UNTAR_.*' {
-    withName: 'UNTAR' {
+
+    withName: 'UNTAR_.*' {
+        ext.args2 = '--no-same-owner'
+    }
+
+    withName: 'UNTAR_.*|BWA_INDEX|BOWTIE2_BUILD|STAR_GENOMEGENERATE' {
         publishDir = [
             path: { "${params.outdir}/genome/index" },
             mode: 'copy',
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
             enabled: params.save_reference
         ]
     }
-    withName: BWA_INDEX {
-        ext.args   = '-a bwtsw'
-        publishDir = [
-            path: { "${params.outdir}/genome/index" },
-            mode: 'copy',
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-        ]
-    }
+
     withName: GFFREAD {
```

Comment:
```suggestion
    withName: 'GFFREAD' {
```

Reply:
Yeah, you are right, I also have found strange behaviours, will do! Actually, sometimes `-resume` is not working when updating an `ext.args` I will check if this is the reason...